### DISCUSSION TITLE
Littlelink-custom for Unraid clears data after an update.

### DISCUSSION THREAD

**Discussion by steRnbear87** [JulianPrieber/llc-docker#2] [Issues]
So I created a LittleLink Custom app for UnRaid Community Appstore from the docker-compose file. My issue is whenever I update the app, it clears my current config and i have to start afresh. Reconfig after every update. Currently using it as it is with sqlite and built in mailserver.

**Comment by lastsamurai26** [comment]
Can you provide us your docker-compose.yml so we can take a look at it

**Comment by steRnbear87** [comment]
![littlelink-custom](https://user-images.githubusercontent.com/46365694/179087303-97163218-36a2-4ff6-bf1e-efa3e3cfe0fe.png)
![littlelink-custom1](https://user-images.githubusercontent.com/46365694/179087304-cfc6cc9c-091a-410a-8d41-9cab97dc1393.png)

**Comment by JulianPrieber** [comment]
Could you try running it without Traefik?

**Comment by erickchung** [comment]
I am experiencing the same problem and I don't have Traefik.
Running only default variables.

**Comment by erickchung** [comment]
I think it's everything (files, configs, sqlite db) gets wiped when the new image gets updated from docker.

So I tried 2 things:
1. I spun up a nginx instance and just dropped the app into www/. I converted everything over to point to my instance of mySQL hosted elsewhere on my network. Like the documentation said, I had to reseed everything and set up llc. Once everything was set up, I backed up the .env and the advanced configs. I assume that this version would be able to just get updated via the regular method.
2. To test out docker, I shut down the nginx server and redeployed the docker image (fresh install) and uploaded the env and the config file that was pointing to mysql. All the links, pages, and settings came back but the uploaded pictures/logos didn't (I assume that it's stored in the file structure). I think it just means that if there is an update to the docker (i.e. wipe), we'll just need to upload those two configs and fix the related images to get it up and running again.

Is there a way to isolate those config files and images to a directory that can be mapped/persisted even between wipes?
Another thought is to add the db connection info in the docker variables and then store the env/adv configs and images in the database...

**Comment by steRnbear87** [comment]
Hey @erickchung I worked with @JulianPrieber and we were able to resolve the issue. The container app in CA appstore has been updated to prevent further updates and only take updates from the built-in updater. Let us know if you still have any issues.

**Comment by joaomelorodrigues** [comment]
Hi, how do we map the links database in a persistent volume? Which directory?

**Comment by bb12489** [comment]
> Hi, how do we map the links database in a persistent volume? Which directory?

I would like to know this as well. I'm trying to run this on my docker swarm cluster.
